.github/workflows/pypi_packaging.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish to PyPi
+
+# Controls when the action will run.
+# Triggered manually
+on:
+  workflow_dispatch:
+    PyPI_Repo:
+        description: 'Where to upload Package'
+        required: true
+        default: 'testpypi'
+        type: choice
+        options:
+        - testpypi
+        - pypi
+    version:
+      description: 'Release Version'
+      required: true
+      type: string
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Setup Env
+        run: |
+          python3 -m pip install --upgrade pip twine build
+
+      - name: Build
+        run: python3 -m build
+
+      - name: Replace version tag in pyproject.toml
+        run: sed -iE "s/version = $PARTITION_COLUMN.*/version = \"${{ inputs.version }}\"/" pyproject.toml
```

Comment:
`$PARTITION_COLUMN` might not needed

Reply:
`sed -iE "s/^\s*version\s*=\s*.*$/version = \"${{ inputs.version }}\"/" pyproject.toml`